aten/src/ATen/native/quantized/cpu/qconv_unpack.cpp

```diff
@@ -134,6 +134,10 @@ class QConvUnpackWeightsInt8 final {
       at::Tensor packed_weight) {
     auto& pack_ptr =
         cpp_custom_type_hack::cast<PackedConvWeightsQnnp>(packed_weight);
+    TORCH_CHECK(
+        pack_ptr.orig_weight.defined(),
+        "Cannot access original weight tensor. "
+        "quantized::conv2d_unpack is currently not supported for qnnpack.");
```

Comment:
I would change this message to work at the level of abstraction that a user is more likely to be thinking at.  Maybe `"Cannot unpack weights.  Call Context::setReleaseOriginalWeights(true) before packing or loading to enable unpacking."`

Reply:
You mean `Context::setReleaseOriginalWeights(false)`?